packages/design-system/src/components/Card/CardAction.tsx

```diff
@@ -0,0 +1,36 @@
+import { styled } from 'styled-components';
+
+import { PropsToTransientProps } from '../../types';
+import { Flex, FlexComponent, FlexProps } from '../Flex';
+
+type CardActionPosition = 'end' | 'start';
+
+type CardActionProps = Omit<FlexProps<'div'>, 'direction' | 'gap' | 'position'> & {
+  position: CardActionPosition;
+};
+
+const CardActionImpl = ({ position, ...restProps }: CardActionProps) => {
+  return <CardAction $position={position} {...restProps} direction="row" gap={2} />;
```

Comment:
Do we intend to prevent direction and gap from being passed in as props?

Reply:
It's omitted from the types, and it'll be overwriten, im not sure what else you'd do?